Разметка внутренней страницы

KladVir opened the pull request:
2ая страница еще не до конца доделала)

---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/1988877/tasks/3)

---
:mortar_board: [Разметка внутренней страницы](https://up.htmlacademy.ru/htmlcss/33/user/1988877/tasks/4)

KladVir committed:
Сделал драфт-версию 1ой и 2ой страницы

2ая страница еще не до конца доделала)

KladVir committed:
Доделал вторую страницу

Внёс несколько изменений в первую страницу + закончил вторую

KladVir commented:
Сделал драфт-версию 1ой и 2ой страницы

rekbrjdf commented on hotels.html:
```diff
@@ -0,0 +1,177 @@
+		</li>
+		
+		
+		<li class="places-list-item">Desert Quail Inn
```
У тебя по всем страница разметка поехала, нет правильных отступов.

rekbrjdf commented on hotels.html:
```diff
@@ -0,0 +1,177 @@
+	<h2 class="visually-hidden">Список найденных гостиниц</h2>
+	<section class="staying-place-list">
+	<span class="">Найдено гостиниц: 38</span>
+	<ol class="places-list">
```
А почему `<ol>` ?

KladVir replied:
Вот тут не понял, в чём ошибка. Ты могла бы поподробней описать?

KladVir replied:
Я думал, что их порядок в выборке будет отсортирован. И если сортировка происходит от, например, меньшего к большему (или наоборот), то порядок будет важен. Такие были мои мысли)

rekbrjdf commented on hotels.html:
```diff
@@ -0,0 +1,177 @@
+	<input type="text" name="from">
+	<input type="text" name="to">
+	0<input type="range" name="price-range" id="price-range" min="0" max="10000" step="1">9000
+	<a class="button button-apply" type="button" href="#">Применить</a>
```
type не указывается в ссылках. И тут больше подходит <button> 
Посмотри тут https://up.htmlacademy.ru/htmlcss/33/demos/6173#16

rekbrjdf commented on hotels.html:
```diff
@@ -0,0 +1,177 @@
+		<button class="button button-list"></button>
+		</li>	
+		<li class="button number-item">3
+		<button class="button button-list"></button>
```
Тут ссылка подойдет <a> https://htmlacademy.ru/blog/boost/frontend/links-or-buttons Почитай)

rekbrjdf commented on index.html:
```diff
@@ -2,11 +2,128 @@
+    <nav class="navigation">
+      <ul class="navigation-list">
+        <li class="navigation-element">
+          <a class="navigation-link" href="main-page">Главная</a></li>
```
Для главной страницы можно в ссылке указать href="/"

rekbrjdf commented on index.html:
```diff
@@ -2,11 +2,128 @@
+        <li class="navigation-element">
+          <a class="navigation-link" href="about-sedona">О Седоне</a></li>
+        <li class="navigation-element">
+          <a class="navigation-link" href="hotels-list">Гостиницы</a></li>
```
![image](https://user-images.githubusercontent.com/18364459/150652521-ee958a1c-7c09-4d4c-a3e3-68696cc023a4.png)
Формат чуть другой ) забыл .html  в конце

rekbrjdf commented on index.html:
```diff
@@ -2,11 +2,128 @@
+   <footer class="page-footer">
+     <section class="footer-socials">
+       <ul class="footer-socials-list">
+          <li class="footer-socials-list-twitter">
```
Не хватает общего класса, объединяющего эти ссылки
Почитай плиз https://up.htmlacademy.ru/htmlcss/33/module/2/item/12
Тут хорошо написано, если что обращайся

KladVir committed:
Внёс правки 23.01.22 20:50(МСК)

Внёс правки, описанные на данной странице в Гит https://github.com/htmlacademy-htmlcss/1988877-sedona-33/pull/2
